tests/subscript_aggregation/README.md

```diff
@@ -15,6 +15,8 @@ Contributions
 |:--------------------------------- |:--------------- |:-------------------------- |:-------- |:----------------------- |
 | test_subscript_aggregation.mdl    | James Houghton  | james.p.houghton@gmail.com | 10/20/15 | Vensim DSS 6.3 for Mac  |
 | output.csv                        | James Houghton  | james.p.houghton@gmail.com | 10/20/15 | Vensim DSS 6.3 for Mac  |
+| `test_subscropt_aggregation.mdl` (shorter)  | Eneko Martin    | eneko.martin.martinez@gmail.com | 11/18/20 | Vensim DSS for Windows 7.3.4 single precision (x32)  |
+| `output.tab ` (shorter)                      | Eneko Martin    | eneko.martin.martinez@gmail.com | 11/18/20 | Vensim DSS for Windows 7.3.4 single precision (x32)  |
 
 Todo:
-Make a case where we sum over (say) dimensions 1 and 3, but not 2
```

Comment:
you mean to work properly in expressions like  `SUM(object[dim1!, dim2, dim3])`?
In that case it should work fine. I can add one of these example to the existing model.

I can also add models of PROD, VMAX and VMIN if you think it is necessary. In any case, the well working of these functions is tested at the end of _unit_test_functions.py_, and the algorithm to write the operating dimensions is the same as the used for the SUM ( defined in vensim2py.py:ExpressionParser:visit_call). Then the good working of PROD VMAX and VMIN is ensured if both the subscript_aggregation model and the tests from _unit_test_functions.py_ pass.

Reply:
Nice. It might make sense to add a test for PROD, VMAX, etc. to the test-models repo (I'll leave it up to you whether to make them separate tests or a single test) just because other projects (SDEverywhere, etc.) are also using the test repo, and it's useful to show functionality between them.

Reply:
Perfect. Now you mention that other projects are using this repo, they should know that at least the Vensim version I am using gives only the value for the first time step for the input constant values. I corrected that on the _test_utils.py:assert_frames_close_ by adding  these lines:
```
 for col in expected.columns:
+    # if for Vensim outputs where constant values are only in the first row
+    if np.isnan(expected[col].values[1:]).all():
+        expected[col] = expected[col].values[0]
```
so probably it will be necessary to add something similar to the other projects that are using this repo. 

It is also possible to add a script to process the _output.tab_ file and fill the missing column values before adding the results to the test-models repo. However, this would make more complicated adding new models.

Reply:
I think I have manually processed the `output.tab` for some of the tests in this repo, but I'm happy to use the Vensim default instead. Let's merge this PR, and open an issue to discuss switching the format for future tests.